src/routes.tsx

```diff
@@ -1,10 +1,14 @@
 import React, { Suspense } from "react";
 import { Route, Switch, useLocation } from "react-router-dom";
+<<<<<<< HEAD
 import { About } from "./pages/About";
 import { CreateGarden } from "./pages/CreateGarden";
 import { GardenView } from "./pages/GardenView";
 import { Home } from "./pages/Home";
 import { MyGardens } from "./pages/MyGardens";
+=======
+import { Loading } from "./components/LoadingWrapper/Loading";
+>>>>>>> a433826e4fdd0445cc89f3f4b068780e8a94947e
 import { useUserState } from "./store/user/useUserState";
```

Comment:
@curtisahuang 
Please check the code if all the conflicts are resolved before making a commit and merging your code